chart/kubeapps/templates/kubeappsapis/deployment.yaml

```diff
@@ -145,18 +148,27 @@ spec:
           {{- if .Values.kubeappsapis.resources }}
           resources: {{- toYaml .Values.kubeappsapis.resources | nindent 12 }}
           {{- end }}
-          {{- if .Values.clusters }}
           volumeMounts:
+          {{- if .Values.clusters }}
             - name: clusters-config
               mountPath: /config
             - name: ca-certs
               mountPath: /etc/additional-clusters-cafiles
           {{- end }}
-      {{- if .Values.clusters }}
+          {{- if .Values.kubeappsapis.pluginConfig }}
+            - name: plugins-config
+              mountPath: /config/kubeapps-apis
```

Comment:
Great, I was wondering whether you could specify a subpath of an existing mount path (`/config`)for the mount path. Glad to know you can.